kuzzleio/kuzzle-device-manager
# feat(measuremodels): add optional local names and unit to measure definitions

## What does this PR do ?

This PR to give the possibility to add friendly names and units internationalized in the measure model definition.
It also adds friendly names and units to default measure models

### How should this be manually tested?
1- On the IOT platform, register a measure via the instance of the app like so :

``` typescript
export const measureDefinition: MeasureDefinition = {
  valuesMappings: { temperature: { type: "float" } },
  valuesDetails: {
    temperature: {
      en: {
        friendlyName: "Temperature",
        unit: "°C",
      },
      fr: {
        friendlyName: "Température",
        unit: "°C",
      },
    },
  },
};


app.plugin.get<DeviceManagerPlugin>('device-manager');

  deviceManager.models.registerMeasure('measureName', measureDefinition);
```
2- On the admin console, in the platform index, models collection, verify that the document corresponding to your measureName has been created with the correct valuesDetails fields.
![image](https://github.com/kuzzleio/kuzzle-device-manager/assets/165774127/d52bd24b-691a-49bb-8e07-b17b3f29ecfa)
